features/steps/allocation_story_steps.py

```diff
@@ -96,12 +96,13 @@ def step_impl(context):
 def step_impl(context):
     for row in context.table:
         provider_alias = context.instance[row['instance_id']]
+        context.client.get('/api/v2/emulate_session/%s'%(row['username']))
```

Comment:
This is cool I didn't know you could do this.

I solved this by using `behaving` personas. See:

https://github.com/julianpistorius/atmosphere/blob/4dfba05dc1ae8f2e1d2b3393bf24fc9d278fbdf9/features/jetstream.features/special_allocation.feature#L58